src/main/java/org/jitsi/jigasi/transcription/Transcriber.java

```diff
@@ -192,8 +217,13 @@ public void start()
     {
         if(State.NOT_STARTED.equals(this.state))
         {
+            logger.debug("Transcriber is now transcribing");
             this.state = State.TRANSCRIBING;
-            executorService = Executors.newSingleThreadExecutor();
+            this.executorService = Executors.newSingleThreadExecutor();
+
+            List<String> names = new ArrayList<>(participants.size());
+            participants.values().forEach((p) -> names.add(p.getName()));
+            this.transcript.started(roomName, names);
```

Comment:
When we list the participants, the ones without a name should be listed with IDs. Perhaps we can handle this by modifying Participant#getName, or handle it here?